src/App.tsx

```diff
@@ -21,14 +34,17 @@ export const App: React.FC = () => {
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {!(todos.length > 0)
```

Comment:
Reverse condition. Better to do `todos.length`, it will be the same as `todos.length > 0`